### PR TITLE
[zh-cn] update css <integer> type

### PR DESCRIPTION
**Pull request by jasonren0403**

### CODE REVIEW

File: files/zh-cn/web/css/integer/index.md [hunk 1]
```diff
@@ -1,50 +1,48 @@
...
 slug: Web/CSS/integer
+l10n:
+  sourceCommit: b4862a953a358b2f08a181e3a4051433cfa86ca0
 ---
 
 {{CSSRef}}
 
-The **`<integer>`** [CSS](/zh-CN/docs/Web/CSS) [data type](/zh-CN/docs/Web/CSS/CSS_Types) is a special type of {{cssxref("number")}} that represents a whole number, whether positive or negative. Integers can be used in numerous CSS properties, such as {{cssxref("column-count")}}, {{cssxref("counter-increment")}}, {{cssxref("grid-column")}}, {{cssxref("grid-row")}}, and {{cssxref("z-index")}}.
-The **`<integer>`** css 数据类型是一种特殊的\<number>类型，它表示一个整数，无论是正数还是负数。整数可以用于许多 css 属性，例如{{cssxref("column-count")}}, {{cssxref("counter-increment")}}, {{cssxref("grid-column")}}, {{cssxref("grid-row")}}, and {{cssxref("z-index")}}.
+**`<integer>`** [CSS](/zh-CN/docs/Web/CSS) [数据类型](/zh-CN/docs/Web/CSS/CSS_Types)是一种特殊的 {{cssxref("number")}} 类型，它表示一个正整数或负整数。整数可用于许多 CSS 属性和描述符，例如{{cssxref("column-count")}}、{{cssxref("counter-increment")}}、{{cssxref("grid-column")}}、{{cssxref("grid-row")}}、{{cssxref("z-index")}} 属性和 {{cssxref("@counter-style/range", "range")}} 描述符。
```
Comment (jasonren0403) [inline files/zh-cn/web/css/integer/index.md:10]:
```suggestion
**`<integer>`** [CSS](/zh-CN/docs/Web/CSS) [数据类型](/zh-CN/docs/Web/CSS/CSS_Types)是一种特殊的 {{cssxref("number")}} 类型，它表示一个整数。整数可用于许多 CSS 属性和描述符，例如{{cssxref("column-count")}}、{{cssxref("counter-increment")}}、{{cssxref("grid-column")}}、{{cssxref("grid-row")}}、{{cssxref("z-index")}} 属性和 {{cssxref("@counter-style/range", "range")}} 描述符。
```

File: files/zh-cn/web/css/integer/index.md [hunk 2]
```diff
@@ -1,50 +1,48 @@
...
-## Interpolation
+## 插值
 
-When animated, values of the `<integer>` data type are interpolated using discrete, whole steps. The calculation is done as if they were real, floating-point numbers; the discrete value is obtained using the [floor function](http://en.wikipedia.org/wiki/Floor_function). The speed of the interpolation is determined by the [timing function](/zh-CN/docs/Web/CSS/single-transition-timing-function) associated with the animation.
-设置动画时，\<integer>数据类型的值将使用离散的完整步骤进行插值。计算过程就像是实数，浮点数一样；离散值是使用 floor 函数得到的。插值的速度由与动画相关联的定时函数确定。
+当使用动画时，{{Glossary("interpolation", "插值法")}}会使用离散的整数步来计算 `<integer>` 数据类型的值。计算方法与实数、浮点数相同；离散值使用 [floor 函数](https://zh.wikipedia.org/wiki/取整函数)获取。插值的速度由与动画关联的[缓动函数](/zh-CN/docs/Web/CSS/easing-function)决定。
 
 ## 示例
 
 ### 合法整型
```
Comment (jasonren0403) [inline files/zh-cn/web/css/integer/index.md:24]:
```suggestion
### 有效的 integer 值
```

File: files/zh-cn/web/css/integer/index.md [hunk 3]
```diff
@@ -1,50 +1,48 @@
...
+```plain example-good
+12          正整数（没有前导 + 号）
++123        正整数（包含前导 + 号）
+-456        负整数
+0           零
++0          有前导 + 号的零
+-0          有前导 - 号的零
 ```
 
 ### 非法整型
```
Comment (jasonren0403) [inline files/zh-cn/web/css/integer/index.md:35]:
```suggestion
### 无效的 integer 值
```

File: files/zh-cn/web/css/integer/index.md [hunk 4]
```diff
@@ -1,50 +1,48 @@
...
 slug: Web/CSS/integer
+l10n:
+  sourceCommit: b4862a953a358b2f08a181e3a4051433cfa86ca0
 ---
 
 {{CSSRef}}
 
-The **`<integer>`** [CSS](/zh-CN/docs/Web/CSS) [data type](/zh-CN/docs/Web/CSS/CSS_Types) is a special type of {{cssxref("number")}} that represents a whole number, whether positive or negative. Integers can be used in numerous CSS properties, such as {{cssxref("column-count")}}, {{cssxref("counter-increment")}}, {{cssxref("grid-column")}}, {{cssxref("grid-row")}}, and {{cssxref("z-index")}}.
-The **`<integer>`** css 数据类型是一种特殊的\<number>类型，它表示一个整数，无论是正数还是负数。整数可以用于许多 css 属性，例如{{cssxref("column-count")}}, {{cssxref("counter-increment")}}, {{cssxref("grid-column")}}, {{cssxref("grid-row")}}, and {{cssxref("z-index")}}.
+**`<integer>`** [CSS](/zh-CN/docs/Web/CSS) [数据类型](/zh-CN/docs/Web/CSS/CSS_Types)是一种特殊的 {{cssxref("number")}} 类型，它表示一个整数。整数可用于许多 CSS 属性和描述符，例如{{cssxref("column-count")}}、{{cssxref("counter-increment")}}、{{cssxref("grid-column")}}、{{cssxref("grid-row")}}、{{cssxref("z-index")}} 属性和 {{cssxref("@counter-style/range", "range")}} 描述符。
```
Comment (skyclouds2001) [inline files/zh-cn/web/css/integer/index.md:10]:
```suggestion
**`<integer>`** [CSS](/zh-CN/docs/Web/CSS) [数据类型](/zh-CN/docs/Web/CSS/CSS_Types)是一种特殊的 {{cssxref("number")}} 类型，它表示一个整数。整数可用于许多 CSS 属性和描述符，例如 {{cssxref("column-count")}}、{{cssxref("counter-increment")}}、{{cssxref("grid-column")}}、{{cssxref("grid-row")}}、{{cssxref("z-index")}} 属性和 {{cssxref("@counter-style/range", "range")}} 描述符。
```

**Review by yin1999 — APPROVED** [review]
Great @jasonren0403

**Pull request merged**